### PR TITLE
add content type tooltips to icons, and remove 'html5' from text

### LINKED ISSUE #1979: tooltips and A11Y for content type indicators
image

The content type icons should have tooltips on hover and a11y labels for screen readers

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

adds tooltips to content icons

image

image

### Reviewer guidance

check it out

### References

fixes #1979

----

### Contributor Checklist

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] Contributor has fully tested the PR manually
- [ ] Screenshots of any front-end changes are in the PR description
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: kolibri/core/assets/src/views/content-icon/index.vue [hunk 1]
```diff
@@ -97,6 +113,23 @@
       colorClass() {
         return `color-${this.colorStyle}`;
       },
+      tooltipText() {
+        const kind = this.kind;
```
Comment (christianmemije) [inline kolibri/core/assets/src/views/content-icon/index.vue:120]: Missing channel, lesson, exam, and user.

Comment (indirectlylit) [reply kolibri/core/assets/src/views/content-icon/index.vue:120]: thanks - updated

File: kolibri/core/assets/src/views/content-icon/index.vue [hunk 2]
```diff
@@ -97,6 +116,29 @@
...
+        } else if (kind === Constants.ContentNodeKinds.DOCUMENT) {
+          return this.$tr('document');
+        } else if (kind === Constants.ContentNodeKinds.HTML5) {
+          return this.$tr('html5');
+        } else if (kind === Constants.ContentNodeKinds.EXAM) {
+          return this.$tr('exam');
+        } else if (kind === Constants.ContentNodeKinds.LESSON) {
+          return this.$tr('lesson');
+        }
+        return this.$tr('item');
```
Comment (christianmemije) [inline kolibri/core/assets/src/views/content-icon/index.vue]: Item would never be used since there is no icon we fallback to?

Comment (indirectlylit) [reply kolibri/core/assets/src/views/content-icon/index.vue]: would you prefer it return an empty string?

File: kolibri/core/assets/src/views/content-icon/index.vue (hunk 1 above)
Comment (christianmemije) [reply kolibri/core/assets/src/views/content-icon/index.vue:120]: Last one is USER

File: kolibri/core/assets/src/views/content-icon/index.vue (hunk 2 above)
Comment (christianmemije) [reply kolibri/core/assets/src/views/content-icon/index.vue]: To avoid adding an extra string to translate, yes.

**Comment by indirectlylit** [comment]
i'm actually not sure if I like it in the coach view .. seems a bit much

**Comment by indirectlylit** [comment]
updated

**Comment by christianmemije** [comment]
Looks good!

**Pull request merged**